docs/CreateAKSClusterForScalarDLAuditor.md

```diff
@@ -0,0 +1,225 @@
+# Create an AKS cluster for ScalarDL Ledger and ScalarDL Auditor
+
+This document explains how to create an Azure Kubernetes Service (AKS) cluster for ScalarDL Ledger and ScalarDL Auditor deployment. For details on how to deploy ScalarDL Ledger and ScalarDL Auditor on an AKS cluster, see [Deploy ScalarDL Ledger and ScalarDL Auditor on AKS](./ManualDeploymentGuideScalarDLAuditorOnAKS.md).
+
+## Before you begin
+
+You must create an AKS cluster based on the following requirements, recommendations, and your project's requirements. For specific details about how to create an AKS cluster, refer to the following official Microsoft documentation based on the tool you use in your environment:
+
+* [Azure CLI](https://learn.microsoft.com/ja-jp/azure/aks/learn/quick-kubernetes-deploy-cli)
+* [PowerShell](https://learn.microsoft.com/ja-jp/azure/aks/learn/quick-kubernetes-deploy-powershell)
+* [Azure portal](https://learn.microsoft.com/ja-jp/azure/aks/learn/quick-kubernetes-deploy-portal)
+
+## Requirements
+
+When deploying ScalarDL Ledger and ScalarDL Auditor, you must:
+
+* Create two AKS clusters by using Kubernetes version 1.21 or higher.
```

Comment:
I think we explain how we manage the clusters with different ADs somewhere, but I forgot where.
Should we link to the doc or should we explain it here?